data_pipeline/scraper/scraper_candidates.py

```diff
@@ -0,0 +1,220 @@
+import os
+import time
+from dircreator import DirCreator
+from time import sleep
+
+import chromedriver_binary  # Adds chromedriver binary to path
+from webdriver_manager.chrome import ChromeDriverManager
+
+from selenium import webdriver
+from selenium.webdriver.common.by import By
+from selenium.webdriver.support.ui import WebDriverWait
+from selenium.webdriver.support import expected_conditions as EC
+
+# Currently un-used
+from selenium.common.exceptions import TimeoutException
+from selenium.webdriver.common.keys import Keys
+from selenium.webdriver.common.action_chains import ActionChains
+
+# ----------------------------------------------------------------------------
+#                               Global Variables
+# ----------------------------------------------------------------------------
+BALLOT_TYPE = 'Office Sought'
+DEFAULT_SLEEP_TIME = 3
+SEARCH_FORM_ADDRESS = 'https://www.southtechhosting.com/SanJoseCity/CampaignDocsWebRetrieval/Search/SearchByElection.aspx'
+
+# Selenium parsing variables
+# Do Not Edit, append only
+SEARCH_TABLE_ROW_ID = 'ctl00_GridContent_gridFilers_DXDataRow'
+SEARCH_TABLE_ROW_XPATH_CONTAINS = '//*[contains(@id,"ctl00_GridContent_gridFilers_DXDataRow")]'
+SEARCH_BUTTON_ID = 'ctl00_DefaultContent_ASPxRoundPanel1_btnFindFilers_CD'
+SEARCH_TABLE_INDEX_CLICKABLE_ID = 'ctl00_GridContent_gridFilers_DXCBtn0Img'
+
+FORM_TABLE_ROW_XPATH_CONTAINS = '//*[contains(@id,"ctl00_DefaultContent_gridFilingForms_DXDataRow")]'
+FORM_TABLE_ROW_ID = 'ctl00_DefaultContent_gridFilingForms_DXDataRow'
+FORM_TABLE_MAIN_TABLE_ID = 'ctl00_DefaultContent_gridFilingForms_DXMainTable'
+
+ERROR_DIALOG_BUTTON_ID = 'ctl00_GridContent_popupCantContinueDialog_Button1'
+PAGE_ENTRY_XPATH = '//a[@class="dxbButton_Glass dxgvCommandColumnItem_Glass dxgv__cci dxbButtonSys"]'
+
+
+class SjcWebsite():
```

Comment:
There should only be one SjcWebsite class, which scraper_candidates and scraper measures should depend on.  It can have functions like GetAllCandidateForms or GetAllMeasureForms.  This will help reduce code duplication.  (I can also help make this change)

Reply:
Great idea!
instead of creating more functions in the SjcWebsite class, I decided to use variables during the Scrape() instantiation.
I think this will help in the future once we've got integrate into the API
let me know what you think